src/ArticleController.h

```diff
@@ -47,6 +47,7 @@
 	NSString * sortColumnIdentifier;
 	BackTrackArray * backtrackArray;
 	BOOL isBacktracking;
+	BOOL shouldPreserveSelectedArticle;
```

Comment:
can we make this a private `@property`?

Reply:
Good point. I'll check what should be made private regarding ArticleController, ArticleListView and UnifiedView.

Reply:
I prefer postponing to a later PR the change to private of ivars and properties.
It requires more work than initially thought and I prefer keeping this PR relatively small. Mixing bug fixing and refactoring is rarely a good idea…